### PR TITLE
Added new parameter

### PR DESCRIPTION
**Pull request by derNiklaas**
See https://github.com/codeoverflow-org/chatoverflow/pull/148

### CODE REVIEW

File: src/main/java/org/codeoverflow/chatoverflow/api/io/parameter/ColorParameter.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package org.codeoverflow.chatoverflow.api.io.parameter;
+
+import org.codeoverflow.chatoverflow.api.IsRequirement;
+
+import java.awt.*;
+
+/**
+ * A {@link Parameter} that is a boolean
```
Comment (hlxid) [inline src/main/java/org/codeoverflow/chatoverflow/api/io/parameter/ColorParameter.java]:
```suggestion
 * A {@link Parameter} that is a Color
```

File: src/main/java/org/codeoverflow/chatoverflow/api/io/parameter/UriParameter.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package org.codeoverflow.chatoverflow.api.io.parameter;
+
+import org.codeoverflow.chatoverflow.api.IsRequirement;
+
+import java.net.URI;
+
+@IsRequirement
+public interface UriParameter extends Parameter<URI> {
+}
```
Comment (hlxid) [inline src/main/java/org/codeoverflow/chatoverflow/api/io/parameter/UriParameter.java]: Please also add a javadoc comment here :wink:

**Review by hlxid — COMMENTED** [review]
Please commit the updated java file of `org.codeoverflow.chatoverflow.api.plugin.configuration.Parameter` which is produced by the generator.

**Review by hlxid — APPROVED** [review]

**Comment by derNiklaas** [comment]
Added / fixed javadoc. Didn't noticed that I missed that.

**Pull request merged**